### PR TITLE
Issue #1: Added unit tests for Configurator.

### PR DESCRIPTION
**Pull request by FridaTveit**

### CODE REVIEW

File: src/test/java/net/sf/javaanpr/configurator/ConfiguratorTest.java [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+    @Test(expected = NumberFormatException.class)
+    public void testSetAndGetIntPropertyWithInvalidInt() {
+        configurator.setStrProperty(propertyName, "not an int");
+        configurator.getIntProperty(propertyName);
+    }
+
+    @Test
+    public void testSetAndGetDoublePropertyWithValidDouble() {
+        double expectedValue = 5.0;
+        double epsilon = 5.96e-08;
```
Comment (oskopek) [inline src/test/java/net/sf/javaanpr/configurator/ConfiguratorTest.java]: Optional: `epsilon` seems to be creeping up in several tests now, do you think we should turn it into a `public static final` "constant" in the `TestUtility` class?

File: src/test/java/net/sf/javaanpr/configurator/ConfiguratorTest.java [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+import static org.junit.Assert.assertEquals;
+
+public class ConfiguratorTest {
+    private Configurator configurator;
+    private final String propertyName = "new property";
+
+    @Before
+    public void setup() {
+        configurator = Configurator.getConfigurator();
```
Comment (oskopek) [inline src/test/java/net/sf/javaanpr/configurator/ConfiguratorTest.java]: The `Configurator`, as it is written now, is pretty bad. The fact that it's a singleton could potentially cause quite a few issues, for example with testing. Another person could come in, create a test that assumes that the "new property" property is empty, because we get a "new" configurator before every test in the setup method, which is not really the case.

I'll merge this, as the test is good, and attempt a fix later this week. If you want, you can create an issue for this.

File: src/test/java/net/sf/javaanpr/configurator/ConfiguratorTest.java (hunk 1 above)
Comment (FridaTveit) [inline src/test/java/net/sf/javaanpr/configurator/ConfiguratorTest.java]: Done in pull request #36 :)
